src/components/common/slider.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface SliderProps {
+	toggleTheme: () => void;
+	isDarkTheme: boolean;
+}
+
+const Slider: React.FC<SliderProps> = ({ toggleTheme, isDarkTheme }) => {
+	const [isToggled, setToggled] = useState(isDarkTheme);
```

Comment:
This component should not have a local state, the slider value should depend on whether the theme is set to dark or not